diff --git a/interpreter/llvm-project/llvm/include/llvm/ADT/iterator.h b/interpreter/llvm-project/llvm/include/llvm/ADT/iterator.h
index 6f0c42f..d792448 100644
--- a/interpreter/llvm-project/llvm/include/llvm/ADT/iterator.h
+++ b/interpreter/llvm-project/llvm/include/llvm/ADT/iterator.h
@@ -142,7 +142,7 @@ public:
         "The '+' operator is only defined for random access iterators.");
     return i + n;
   }
-  DerivedT operator-(DifferenceTypeT n) const {
+  DerivedT operator-(DifferenceTypeT n) const requires (bool(IsRandomAccess)) {
     static_assert(
         IsRandomAccess,
         "The '-' operator is only defined for random access iterators.");
@@ -267,7 +267,7 @@ public:
     return *static_cast<DerivedT *>(this);
   }
   using BaseT::operator-;
-  difference_type operator-(const DerivedT &RHS) const {
+  difference_type operator-(const DerivedT &RHS) const requires (bool(BaseT::IsRandomAccess))  {
     static_assert(
         BaseT::IsRandomAccess,
         "The '-' operator is only defined for random access iterators.");
