diff --git a/contrib/stdhep/stdhep_mcfio.c b/contrib/stdhep/stdhep_mcfio.c
index 1aa75d2..82946ec 100644
--- a/contrib/stdhep/stdhep_mcfio.c
+++ b/contrib/stdhep/stdhep_mcfio.c
@@ -62,13 +62,13 @@ struct stdstr stdstr_;
 struct stdhd1 stdhd1_;
 struct stdhd2 stdhd2_;
 
-extern int xdr_stdhep_();
-extern int xdr_stdhep_multi_();
-extern int xdr_stdhep_4_();
-extern int xdr_stdhep_4_multi_();
-extern int xdr_stdhep_cm1_();
-extern int xdr_hepeup_();
-extern int xdr_heprup_();
+extern int xdr_stdhep_(XDR *, int *, int *, char **);
+extern int xdr_stdhep_multi_(XDR *, int *, int *, char **);
+extern int xdr_stdhep_4_(XDR *, int *, int *, char **);
+extern int xdr_stdhep_4_multi_(XDR *, int *, int *, char **);
+extern int xdr_stdhep_cm1_(XDR *, int *, int *, char **);
+extern int xdr_hepeup_(XDR *, int *, int *, char **);
+extern int xdr_heprup_(XDR *, int *, int *, char **);
 
 int StdHepXdrReadInit(char *filename, int ntries, int ist)
 {
